addons/website/views/website_templates.xml

```diff
@@ -318,9 +318,9 @@
             <!-- Brand -->
             <t t-call="website.placeholder_header_brand"/>
             <ul class="o_header_mobile_buttons_wrap navbar-nav flex-row align-items-center gap-2 mb-0">
-                <li>
+                <li class="o_not_editable">
```

Comment:
Some other buttons are added by other apps (wishlist, cart btn, etc.) I don't want to take the risk to break something.
But maybe it's something to understand 🤔💭 ... why does the overlay go on the hamburger buttons and not on the others .

Reply:
@sobo-odoo In fact, the `<li>` of the "hamburger" toggler is editable becasue it's a view.
See `editableElements($wrapwrap) {` in  `wysiwyg_adapter.js`.
I'm not 100% sure if all elements in `o_header_mobile_buttons_wrap` need to be not editable (custo ?) that's why I would leave it like.. What do you think ?
Except that this PR is ready ...

Reply:
I agree with you :+1: 

A last detail about this, now that we are in 17.0, maybe make sure we can change this directly in the template :wink: 
Normally it is okay, because:
- it is an addition of something (so not a deletion) so it doesn't break anything
- the already created databases won't have the fix, but since it is not a real bug (just a nice to have) I think it is okay, they will have it when they run -u 

But just to be sure it is okay, maybe ask ?

Reply:
After discussion, we will only add the class in stable and then remove the other one in master.